### ISSUE TITLE
Get-HPOVOSDeploymentServer returns nothing

### ISSUE THREAD

**Issue by sinelshchikov** [HewlettPackard/POSH-HPOneView#280]
In the latest release the Get-HPOVOSDeploymentServer returns nothing.

I found the old Get-HPOVOSDeploymentServer function from some old POSHHPOV that works replaced it in the latest release , and it is working.

I am attaching the "working" function for you to check.

[Get-HPOVOSDeploymentServer.txt](https://github.com/HewlettPackard/POSH-HPOneView/files/1113467/Get-HPOVOSDeploymentServer.txt)

### Expected Behavior
```
PS C:\Users\Administrator> Get-HPOVOSDeploymentServer

Appliance  Name Status State     PrimaryIPv4

10.1.0.101 is   OK     Connected 10.1.0.80
```

### Actual Behavior

```
PS C:\Users\Administrator> Get-HPOVOSDeploymentServer
PS C:\Users\Administrator>
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): 3.0.1358.2596
HPE OneView Appliance Version:  3.10.04-0299553
Output from `$PSVersionTable` on your Windows Host:
``` PowerShell
Name                           Value
----                           -----
PSVersion                      4.0
WSManStackVersion              3.0
SerializationVersion           1.1.0.1
CLRVersion                     4.0.30319.42000
BuildVersion                   6.3.9600.17400
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0}
PSRemotingProtocolVersion      2.2
```

**Comment by ChrisLynchHPE** [comment]
I will need you to provide me a capture of the Cmdlet running with the `-Verbose` switch added at the end.  Please capture the output and post it to this issue.

**Comment by ChrisLynchHPE** [comment]
I found the issue.  It was caused by a typo in the return object.

**Comment by sinelshchikov** [comment]
Sorry for the delay, Chris.
Do you still need me to run it with -verbose ?

**Comment by ChrisLynchHPE** [comment]
No.  I know exactly what is causing this, and have it fixed for the next release.

**Comment by ChrisLynchHPE** [comment]
Sorry to be late with the notification, but this issue was fixed a while ago, and can be found in [Release 3.0.1358.2596](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1358.2596)

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1358.2596](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1358.2596).

However, with the 3.00 library no longer supported, I am closing this issue.  Please upgrade your HPE OneView appliance (if not already) and OneView PowerShell library.
